bmwqemu.pm

```diff
@@ -64,8 +64,8 @@ our $direct_output;
 # the kraxel.org nightly packages, third is Fedora's edk2-ovmf package,
 # fourth is Debian's ovmf package.
 our @ovmf_locations = (
-    '/usr/share/qemu/ovmf-x86_64-ms.bin', '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd',
-    '/usr/share/edk2/ovmf/OVMF_CODE.fd',  '/usr/share/OVMF/OVMF_CODE.fd'
+    '/usr/share/qemu/ovmf-x86_64-ms-code.bin', '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd',
```

Comment:
It is possible that this file is present on other architectures as x86 can be emulated.

Reply:
yes, as was the case in before. I did not change this and also I don't think of it as an imminent problem. Did you just want to mention this? Do you see a problem? Do you have a suggestion?

Reply:
This was only used on x86 before, at least in this part of the code. Because you now use it to set the pflash code variable in general it could be used in more scenarios. I would suggest using the arch variable to generate the file name somehow.

Reply:
my bad. I did not remember over the weekend that I removed the "if x86_64" part. Will change that after crosschecking other architectures.

Reply:
Same as before the according code is only called for x86_64 now as I figured for other architectures the firmware can not be easily discovered, e.g. aarch64 would be `aavmf-aarch64-ms-code.bin` so not a simple s/x86_64/aarch64/